components/create_team/components/team_url/team_url.tsx

```diff
@@ -140,7 +152,9 @@ export default class TeamUrl extends React.PureComponent {
             return;
         }
 
-        const {data, error} = await createTeam(teamSignup.team);
+        const createTeamData: { data: Team; error: any } = await createTeam(teamSignup.team);
```

Comment:
`error` here can be `Client4Error` I believe.
Actually I'm not sure you even need to declare the type here.

Reply:
It causes 
>  error TS2339: Property 'message' does not exist on type 'Client4Error'.Property 'message' does not exist on type 'ErrorInvalidResponse'.
issue. The same is if type of error is set to Client4Error. I remember I used any because of this.